docs/language/builtins.rst

```diff
@@ -237,15 +237,15 @@ ABI encodes the arguments to bytes. Any number of arguments can be provided.
 
 On Substrate, foo will be ``hex"f100"``. On Ethereum this will be ``hex"00000000000000000000000000000000000000000000000000000000000000f1"``.
 
-abi.encodeWithSelector(bytes4 selector, ...)
+abi.encodeWithSelector(selector, ...)
 ++++++++++++++++++++++++++++++++++++++++++++
 
 ABI encodes the arguments with the function selector first. After the selector, any number of arguments
 can be provided.
 
 .. code-block:: solidity
 
-    bytes foo = abi.encodeWithSelector(hex"01020304", uint16(0xff00), "ABCD");
+    bytes foo = abi.encodeWithSelector(hex"0102030405060708", uint16(0xff00), "ABCD");
```

Comment:
This should be marked as an Solana example, just like the one below is marked Substrate